src/main/frontend/db/query_dsl.cljs

```diff
@@ -489,6 +532,7 @@
                                               result)]
                                  (add-bindings! form result)))]
                   {:query result
+                   :rules (mapv rules/query-dsl-rules rules)
```

Comment:
We dynamically build rules to provide the minimum amount of rules needed for a query